build-system/test-configs/forbidden-terms.js

```diff
@@ -64,15 +65,15 @@ const forbiddenTermsGlobal = {
   'DO NOT SUBMIT': {
     checkProse: true,
   },
-  'whitelist|white-list': {
+  'white[-\\s]*list': {
     message: 'Please use the term allowlist instead',
     checkProse: true,
   },
-  'blacklist|black-list': {
+  'black[-\\s]*list': {
     message: 'Please use the term denylist instead',
     checkProse: true,
   },
-  'grandfather|grandfathered': {
+  'grandfather': {
```

Comment:
non-actionable comment: I guess there's no grep-able way to detect the context in which this is being used so we can allow "grandfather clock" but forbid other usage?

Reply:
I guess not 😬